api/BaseAudioContext.json

```diff
@@ -1170,47 +1401,106 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "disable_normalisation": {
```

Comment:
This should be `disableNormalization`, per the spec and the MDN page.

Reply:
I think that it's ok as it is now. In the rendered table it will show up as what's written in the description field, and I wrote "disable_normalization" because I saw it used as a naming convention for notes on a subfeature that may need to take up row on their own (such as "supported_on_workers").

Reply:
Well I could clarify it a little actually